src/article/styles/right-column.scss

```diff
@@ -6,7 +6,7 @@
 @mixin _right-column-section() {
   border-bottom: settings.$border;
   margin-top: 36px;
-  padding-bottom: 36px;
+  padding-bottom: 24px;
```

Comment:
I think this padding change makes the spacing above the column separator (the padding + border bottom) and below it (the margin) seem uneven now?

example: 
<img width="271" alt="spacing" src="https://user-images.githubusercontent.com/375812/177501023-3b124edd-9cca-4e6e-bfc2-b352b1508251.png">

Reply:
@scottaubrey I spoke to @chris-huggins about this yesterday and he requested the padding to be 24px. All the spacing should be set at 24px.